anchor/eth/src/abi/ssv_contract.json

```diff
@@ -0,0 +1,2125 @@
+[
```

Comment:
From [here](https://etherscan.io/address/0xa2f1dadbb9e836b7ec47330ff9e5947d2f36fc35#code). I believe the one in the repo is old but the only thing that matters is the event signatures which they cannot change. I may also just remove the entire json and just define all the events with `sol!` since that is all that matters anyways.